Assignment5

IanEGoss opened the pull request.

IanEGoss committed:
Add files via upload

IanEGoss committed:
Merge branch 'IntelliTect-Samples:main' into main

IanEGoss committed:
Merge branch 'IntelliTect-Samples:main' into main

GrantErickson committed:
Auth Tables

GrantErickson committed:
Tokens and other auth

GrantErickson committed:
Update README.md

GrantErickson committed:
Updates to Auth on client side.

meghanmae committed:
Update Final.md

GrantErickson committed:
Minor class updates.

GrantErickson committed:
Update Lectures.md

GrantErickson committed:
Merge pull request #122 from IntelliTect-Samples/Auth

Auth merge into Main

GrantErickson committed:
Add testing to API action

GrantErickson committed:
Route guards

GrantErickson committed:
Class work

GrantErickson committed:
Cleanup

GrantErickson committed:
Basic list working and initial APIs

GrantErickson committed:
All but policies

GrantErickson committed:
Run all builds and only deploy when pushing to master.

GrantErickson committed:
Update azure-app-service-api.yml

GrantErickson committed:
Update azure-static-web-app.yml

GrantErickson committed:
Fix WOTD stats unit tests and word of the day code.

GrantErickson committed:
Update azure-app-service-api.yml

GrantErickson committed:
Update azure-app-service-api.yml

GrantErickson committed:
Merge pull request #139 from IntelliTect-Samples/Assignment-5-Done

Assignment 5 done

GrantErickson committed:
Pagination and Policies

IanEGoss committed:
Merge branch 'IntelliTect-Samples:main' into Assignment5

meghanmae commented on Wordle.Api/Wordle.Api/Controllers/WordController.cs:
```diff
@@ -30,13 +33,15 @@ public async Task<IEnumerable<Word>> GetManyWords(int? count)
 
         [HttpPost]
-        public async Task<Word> AddWord(string newWord, bool isCommon)
+        public async Task<Response<Word>> AddWord(string newWord, bool isCommon)
```
This should have an auth policy on it or be removed in favor of your other add word method

meghanmae reviewed (commented):
Looks like your code was never deployed to your static app?

IanEGoss commented:
I cant get anything to work, I deleted everything and remade the resource group and nothing will compile or run unless it comes directly from the CSCD main branch, if I make any changes to the file at all it stops running, and when I deleted the resource group and remade it nothing works at all now, anyways it was an okay class but I cant get anything to work so gg have a nice summer.
________________________________
From: Meghan ***@***.***>
Sent: Thursday, June 15, 2023 6:42 AM
To: IntelliTect-Samples/EWU-CSCD379-2023-Spring ***@***.***>
Cc: Goss, Ian ***@***.***>; Author ***@***.***>
Subject: Re: [IntelliTect-Samples/EWU-CSCD379-2023-Spring] Assignment5 (PR #144)


Closed #144<https://github.com/IntelliTect-Samples/EWU-CSCD379-2023-Spring/pull/144>.

—
Reply to this email directly, view it on GitHub<https://github.com/IntelliTect-Samples/EWU-CSCD379-2023-Spring/pull/144#event-9540699153>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AXILATNSH2UR5N4LTN4TLELXLMGKXANCNFSM6AAAAAAY73MMNM>.
You are receiving this because you authored the thread.Message ID: ***@***.***>


CAUTION This email originated from outside your organization. Please exercise caution when clicking any links or opening attachments.
